core/gesture.js

```diff
@@ -764,7 +764,7 @@ Blockly.Gesture.prototype.doWorkspaceClick_ = function(e) {
   var ws = this.creatorWorkspace_;
   ws.cursor.hide();
   if (e.shiftKey) {
-    var screenCoord = new goog.math.Coordinate(e.clientX, e.clientY);
```

Comment:
I'd love to know how this one slipped through my efforts last month...
Ah, @alschmiedt added it last week.  It's not in master.